tests/modules/sightings/test_models.py

```diff
@@ -95,3 +98,79 @@ def test_sighting_ensure_no_duplicate_encounters(db):
 
     logging.info(test_sighting.get_encounters())
     logging.info(test_encounter.get_sighting())
+
+
+def test_sighting_jobs(db, request):
+    from app.modules.sightings.models import Sighting, SightingStage
+
+    example_ia_configs = {
+        'config1': {
+            'matchingSetDataOwners': ['user1'],
+            'algorithms': {'algorithm_id': 'algorithm1'},
+        },
+        'config2': {
+            'matchingSetDataOwners': ['user2'],
+            'algorithms': {'algorithm_id': 'algorithm2'},
+        },
+    }
+    sighting1 = Sighting(
+        stage=SightingStage.identification,
+        ia_configs=example_ia_configs,
+    )
+    sighting2 = Sighting(
+        stage=SightingStage.identification,
+        ia_configs=example_ia_configs,
+    )
+    with db.session.begin():
+        db.session.add(sighting1)
+        db.session.add(sighting2)
+    request.addfinalizer(sighting1.delete)
+    request.addfinalizer(sighting2.delete)
+    mock_acm = mock.patch('app.modules.sightings.models.current_app.acm')
+    mock_acm.start()
+    request.addfinalizer(mock_acm.stop)
+
+    job_id1 = uuid.uuid4()
+    job_id2 = uuid.uuid4()
+    job_id3 = uuid.uuid4()
+    job_ids = [job_id3, job_id2, job_id1]
+    mock_uuid = mock.patch(
+        'app.modules.sightings.models.uuid.uuid4', side_effect=job_ids.pop
+    )
+    mock_uuid.start()
+    request.addfinalizer(mock_uuid.stop)
+
+    now = datetime.datetime(2021, 7, 7, 23, 43, 55)
+    with mock.patch('datetime.datetime') as mock_datetime:
+        mock_datetime.utcnow.return_value = now
+        with mock.patch.object(sighting1, 'build_identification_request'):
```

Comment:
Baffled here as to the effect of this mock, what does this patched build_identification_request now return? If it's {} then the jobs wouldn't be created

Reply:
Since I didn't specify `side_effect` or `return_value`, `sighting1.build_identification_request` returns a mock object by default.